### DISCUSSION TITLE
¿Como recibir los mensajes con esta libreria?

### DISCUSSION THREAD

**Discussion by diegomottadev** [netflie/whatsapp-cloud-api#29] [General]
Hola veo que el package tiene solo para enviar datos, enviar texto y enviar documentos, pero si quiero recibir msj para poder dar un tratamiento a las respuestas, tenes hecho un webhook en php?

**Comment by aalbarca** [comment]
Hola @diegomottadev 

Para recibir mensajes debes configurar tu propio webhook -> https://developers.facebook.com/docs/whatsapp/cloud-api/guides/set-up-webhooks

Saludos.
